### PR TITLE
[ES] .../Beatmap Nominators/Becoming a Beatmap Nominator

### PR DESCRIPTION
**Pull request by Lalo611**
Creation of `es.md` for `Beatmap Nominator/Becoming a Beatmap Nominator`.

### CODE REVIEW

File: wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/es.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- Identificación de problemas aislados como de temática general
+
+### Cómo el puntaje mínimo es calculadora
+
+- Esto depende del total de `Puntaje de Modeo` que el postulante ha alcanzado durante los últimos meses. Este valor se determina de acuerdo a  la siguiente fórmula:
+
+![Fórmula del Puntaje de Modeo](/wiki/shared/Modscore_new_wiki.png "Fórmula del Puntaje de Modeo")
+
+`m` = Número mínimo esperado de beatmaps modeados individualmente en un mes
+`M` = Número total de beatmaps modeados individualmente en un mes
```
Comment (Lalo611) [inline wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/es.md:41]: I don't know why, but on rich diff it shows that it's not like this...

Comment (MegaApplePi) [reply wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/es.md:41]: Line 40 in the English version has a line break (indicated by the 2 trailing spaces on line 40). For your PR, please use a trailing backslash instead.

```suggestion
`m` = Número mínimo esperado de beatmaps modeados individualmente en un mes\
`M` = Número total de beatmaps modeados individualmente en un mes
```

**Comment by peppy** [comment]
what is going on here..

**Comment by Lalo611** [comment]
I was playing with the cmd to get to know it and weird things happened :(

**Pull request closed without merging**